### ISSUE TITLE
Why the loss function of Discriminator is MSEloss but not BCEloss?

### ISSUE THREAD

**Issue by nongkeyuandebaocai** [junyanz/pytorch-CycleGAN-and-pix2pix#1206]

**Comment by Mr-Optimistic** [comment]
Hi Please refer this paper: https://arxiv.org/abs/1611.04076
